frontend/simple/main.js

```diff
@@ -58,7 +58,7 @@ async function startApp () {
     },
     computed: {
       showSidebar () {
-        return this.$store.state.loggedIn
+        return this.$store.state.loggedIn // NOTE return true to test on a iPhone
```

Comment:
What is this comment mean? Is this something that you can address in this PR? (Or does it need an issue created for it?)

Reply:
Oh sorry, I tested this on iPhone and to be able to test it (since we can't logIn on iPhone using localhost), I had to change some variables like this one and router guards. I changed back the variable but forgot to remove the NOTE. Removed now :)